Modify Rule S5725: Improved its text (#1447)
SonarSource/rspec: rules/S5725/message.adoc

=== Message

> Using artifacts without integrity checks exposes the end-user to various attacks. Make sure it is safe here.

Reviewer: I think "exposes the end-user to various attacks" a somehow a synonym for "is security sensitive". Except that, to me, "various attacks" might be a bit frightening compared to the difficulty to set up a successful supply chain attack.
```suggestion
Using artifacts without integrity checks is security sensitive. Make sure it is safe here.
```
```suggestion
Make sure using artifacts without integrity checks is safe here.
```
